src/Zope2/Startup/tests/test_schema.py

```diff
@@ -43,26 +42,23 @@ class WSGIStartupTestCase(unittest.TestCase):
 
     def setUp(self):
         self.default_encoding = ZPublisher.HTTPRequest.default_encoding
+        self.TEMPNAME = tempfile.mkdtemp()
+        self.TEMPVAR = os.path.join(self.TEMPNAME, "var")
+        os.mkdir(self.TEMPVAR)
 
     def tearDown(self):
+        shutil.rmtree(self.TEMPNAME)
```

Comment:
Using a context manager will save you from taking care of removing the created temp folder

Reply:
Like I said in my other comment, I am not storing the folder itself. The documentation states that the user is responsible for deletion, anyway.